mingw-w64-python-tinycss2/PKGBUILD

```
@@ -0,0 +1,38 @@
+# Contributor: Raed Rizqie <raed.rizqie@gmail.com>
+
+_realname=tinycss2
+pkgbase=mingw-w64-python-${_realname}
+pkgname=${MINGW_PACKAGE_PREFIX}-python-${_realname}
+pkgver=1.1.0
+pkgrel=1
+pkgdesc='A tiny CSS parser (mingw-w64)'
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+license=('BSD')
+url='https://github.com/Kozea/tinycss2'
+makedepends=(
+    ${MINGW_PACKAGE_PREFIX}-python
+    ${MINGW_PACKAGE_PREFIX}-python-setuptools)
+source=("https://files.pythonhosted.org/packages/ce/d3/ece7a98d5826bd134e269a3a3030153d30482194fca71d95a3041812aab8/tinycss2-1.1.0.tar.gz")
```

Can you change this so that it updates based on version? ie.
```suggestion
source=("${_realname}-${pkgver}.tar.gz::https://pypi.org/packages/source/${_realname::1}/${_realname}/${_realname}-${pkgver}.tar.gz")
```